rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cat.health/10_basic.yml

```diff
@@ -1,32 +1,45 @@
 ---
-"Help":
+"Help - with unassigned_pri_shard_count":
+  - requires:
+      capabilities:
+        - method: GET
+          path: /_cluster/health
+          capabilities: [ unassigned_pri_shard_count ]
+      test_runner_features: capabilities
+      reason: Capability required to run test
   - do:
       cat.health:
         help: true
 
   - match:
       $body: |
-               /^  epoch         .+ \n
-                   timestamp     .+ \n
-                   cluster       .+ \n
-                   status        .+ \n
-                   node.total    .+ \n
-                   node.data     .+ \n
-                   shards        .+ \n
-                   pri           .+ \n
-                   relo          .+ \n
-                   init          .+ \n
-                   unassign      .+ \n
```

Comment:
I won't ask you to revert these changes in this PR, but I personally try to avoid making formatting changes in files like these because there is no enforced format here - and reducing the number of changed lines makes it easier to review (not just the PR, but also the commit in case someone wants to look at this commit in the future).